sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/tests/ScenarioTests/AuthenticationFuntionalTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.RedisEnterprise.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.RedisEnterprise.Tests.ScenarioTests
+{
+    public class AuthenticationFuntionalTests : RedisEnterpriseManagementTestBase
+    {
+        public AuthenticationFuntionalTests(bool isAsync)
+                   : base(isAsync, RecordedTestMode.Record)
```

Comment:
do not leave `RecordedTestMode.Record` in the PR. It should be determined by the environment variable, otherwise your test cases won't run correctly in pipeline.
```suggestion
                   : base(isAsync)
```

Reply:
I have updated this change